packages/icon/src/spectrum-icon.css

```diff
@@ -11,54 +11,62 @@ governing permissions and limitations under the License.
 */
 
 /* THIS FILE IS MACHINE GENERATED. DO NOT EDIT */
-:host([size='xxs']) {
+:host {
+    pointer-events: none;
+    transform: none;
+    transform: var(--spectrum-icon-transform, none);
+    inline-size: var(
+        --mod-icon-inline-size,
+        var(--mod-icon-size, var(--spectrum-icon-size))
+    );
+    block-size: var(
+        --mod-icon-block-size,
+        var(--mod-icon-size, var(--spectrum-icon-size))
+    );
+    color: inherit;
+    color: var(--mod-icon-color, inherit);
+    fill: currentColor;
+    display: inline-block;
+}
+
+:host([size='xxs']),
+:host([size='xxs']) .spectrum-UIIcon {
     --spectrum-icon-size: var(--spectrum-icon-size-xxs);
 }
 
-:host([size='xs']) {
+:host([size='xs']),
+:host([size='xs']) .spectrum-UIIcon {
     --spectrum-icon-size: var(--spectrum-icon-size-xs);
 }
 
-:host([size='s']) {
+:host([size='s']),
+:host([size='s']) .spectrum-UIIcon {
     --spectrum-icon-size: var(--spectrum-icon-size-s);
 }
 
 :host,
-:host([size='m']) {
+:host([size='m']),
+:host([size='m']) .spectrum-UIIcon {
     --spectrum-icon-size: var(--spectrum-icon-size-m);
 }
 
-:host([size='l']) {
+:host([size='l']),
+:host([size='l']) .spectrum-UIIcon {
     --spectrum-icon-size: var(--spectrum-icon-size-l);
 }
 
-:host([size='xl']) {
+:host([size='xl']),
+:host([size='xl']) .spectrum-UIIcon {
     --spectrum-icon-size: var(--spectrum-icon-size-xl);
 }
 
-:host([size='xxl']) {
+:host([size='xxl']),
+:host([size='xxl']) .spectrum-UIIcon {
     --spectrum-icon-size: var(--spectrum-icon-size-xxl);
 }
 
-:host {
-    pointer-events: none;
-    transform: none;
-    transform: var(--spectrum-icon-transform, none);
-    color: inherit;
-    color: var(--mod-icon-color, inherit);
-    fill: currentColor;
-    display: inline-block;
-}
-
-:host {
-    inline-size: var(
-        --mod-icon-inline-size,
-        var(--mod-icon-size, var(--spectrum-icon-size))
-    );
-    block-size: var(
-        --mod-icon-block-size,
-        var(--mod-icon-size, var(--spectrum-icon-size))
-    );
+:host(:not(:root)) {
+    overflow: hidden;
```

Comment:
Hmm, this change introduced clipped icons in Safari. Can you justify why this code was added?